website/views.py

```diff
@@ -451,71 +544,80 @@ def vote_post(request):
         else:
             return HttpResponse("Error: Bad Action.")
 
-        num_votes = cur_post.userUpVotes.count() - cur_post.userDownVotes.count()
+        num_votes = cur_post.userUpVotes.count() - cur_post.userDownVotes\
+                                                           .count()
         cur_post.num_votes = num_votes
         cur_post.save()
         return HttpResponse(num_votes)
             
     else:
         return HttpResponse(initial_votes)
-    
+
+
 # return number of votes and initial votes
```

Comment:
This should be a docstrings. Please refer to Python docs and add this info as a docstring for this function